code/go/0chain.net/smartcontract/faucetsc/handler.go

```diff
@@ -52,7 +55,8 @@ func (fc *FaucetSmartContract) globalPerodicLimit(ctx context.Context, params ur
 func (fc *FaucetSmartContract) pourAmount(ctx context.Context, params url.Values, balances c_state.StateContextI) (interface{}, error) {
 	gn, err := fc.getGlobalNode(balances)
 	if err != nil {
-		return nil, common.NewError("failed to get limits", "global node does not exist")
+		err := smartcontract.NewError(smartcontract.FailRetrievingLimitsErr, "global node does not exist")
```

Comment:
We could use a different error "more related to pourAmount" instead of FailRetrievingLimitsErr.

Reply:
Done.